module/Finna/src/Finna/RecordDriver/SolrLido.php

```diff
@@ -3015,4 +3015,26 @@ protected function getPlaceIDType(\SimpleXMLElement $placeID): string
         }
         return $type;
     }
+
+    /**
+     * Get actor's primary ID
+     *
+     * @param \SimpleXmlElement $actor element
```

Comment:
Siellä on muualla vähän epätavalliseen tyyliin noita parametrien kommentteja, mutta laittaisin näin.
```suggestion
     * @param \SimpleXMLElement $actor Actor
```